perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -570,9 +570,17 @@ sub bin_services_for_address {
             my $ref = join(',', @{$_->{Ref}{Value}{anyType}});
             my $completed = construct_bin_date($_->{CompletedDate});
             my $state = $_->{State}{Name} || '';
+
+            my $resolution_id = $_->{Resolution}{Ref}{Value}{anyType} || 'x';
```

Comment:
Is the `x` a placeholder so the subsequent `me.external_status_code` fails? If so, would it be better/clearer to just skip the template lookup altogether if `$_->{Resolution}{Ref}{Value}{anyType}` doesn't exist?

Reply:
Not sure why I did this! Will do.